OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/createDomainAndStart.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+# #
+# # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
```

Comment:
Please change all license URLs to use `https://` and remove the double `# #` at the beginning of each line.

Reply:
done